src/utils/request/parseBody.ts

```diff
@@ -17,15 +17,15 @@ export function parseBody(body?: MockedRequest['body'], headers?: Headers) {
   // parse it into an object.
   const hasMultipartContent = contentType?.startsWith('multipart/form-data')
   if (hasMultipartContent && typeof body !== 'object') {
-    return parseMultipartData(body, headers) || body
+    return parseMultipartData(body.toString(), headers) || body
   }
 
   // If the intercepted request's body has a JSON Content-Type
   // parse it into an object.
   const hasJsonContent = contentType?.includes('json')
 
   if (hasJsonContent && typeof body !== 'object') {
-    return jsonParse(body) || body
+    return jsonParse(body.toString()) || body
```

Comment:
This is strange. `JSON.parse` accepts numbers just fine but it's TS call signature is limited to accept only `string` input. I feel it's a slight overcomplication to convert everything to a string given `JSON.parse` can handle numbers just fine. I wonder what's your opinion on this?